picard/track.py

```diff
@@ -308,48 +312,60 @@ def _customize_metadata(self):
         if config.setting['convert_punctuation']:
             tm.apply_func(asciipunct)
 
+    @staticmethod
+    def _genres_to_metadata(genres, limit=None, minusage=0, filters='', join_with=None):
+        if limit is not None and limit < 1:
+            return []
+
+        # Ignore tags with zero or lower score
+        genres = +genres
+        if not genres:
+            return []
+
+        # Find most common genres
+        most_common_genres = genres.most_common(limit)
+        topcount = most_common_genres[0][1]
+
+        # Filter by name and usage
+        genres_filter = TagGenreFilter(filters)
+        genres_list = []
+        for name, count in genres_filter.filter(most_common_genres):
+            percent = 100 * count // topcount
+            if percent < minusage:
+                break
+            name = _TRANSLATE_TAGS.get(name, name.title())
+            genres_list.append(name)
```

Comment:
As the following code block is very small I would do this as 

```
if percent >= minusage:
    name = _TRANSLATE_TAGS.get(name, name.title())
    genres_list.append(name)
``` 

I think that reads clearer. In general I think we should avoid the early `break`  / `continue`  calls except for cases where it clearly makes sense or avoids overly deep indentation.

Reply:
Nope, that would be different.
The list we iterate in is sorted, the break actually reduces the number of iterations, inverting the test and removing it would lead to the same result, but with (possibly) more iterations.

Reply:
Righ, I misread that.